### DISCUSSION TITLE
加载时间变得很长

### DISCUSSION THREAD

**Discussion by feiyu233** [LocalizeLimbusCompany/LocalizeLimbusCompany#166] [问答 | Q&A]
是否为月季官方优化问题，有时候能卡在加载页面两分钟
[Log-20230826.log](https://github.com/LocalizeLimbusCompany/LocalizeLimbusCompany/files/12445459/Log-20230826.log)

**Comment by ZengXiaoPi** [comment]
姑且认定为你的电脑配置太差了
~~而且你上传安装器的日志干什么~~
